moped-database/migrations/1707416196011_add_council_districts_to_project_view/down.sql

```diff
@@ -0,0 +1,519 @@
+-- no "IF EXISTS" because if it doesn't exist, we have bigger problems and we want this to bail
+DROP VIEW public.project_list_view CASCADE;
```

Comment:
Here, and in the up migration, you'll see that I need to drop the view `CASCADE`, and it cascades onto the AGOL view that depends on the project list view. This is why the migrations have to reestablish both views.